contracts/README.md

```diff
@@ -26,6 +26,7 @@ score = deadline - now
 There are two built-in incentives:
 * playing for the score: If you clicked in the 10th second of TheButton's life, which is set for example to 900 blocks, you get rewarded based on the score of 900-10=890 (and the button's life now will end at block 910).
 * playing to be ThePressiah: the last player to click gets 50% of the total reward pool.
+
 ## BackToTheFuture
 
 In this scenario the rewards are reversed - players get rewarded for extending the button's life further into the future, i.e.:
```

Comment:
Not related to this PR, but maybe we can specify a minimal `cargo contract` version in this readme? E.g. I was having an old version

```
The contract was built in RELEASE mode.

Your contract artifacts are ready. You can find them in:
/home/marcin/git/aleph-node/contracts/back_to_the_future/target/ink

  - back_to_the_future.contract (code + metadata)
  - back_to_the_future.wasm (the contract's code)
  - metadata.json (the contract's metadata)
error: Found argument 'instantiate' which wasn't expected, or isn't valid in this context

marcin@marcin-Latitude-3520 ~/g/aleph-node (blue-button) [1]> cargo contract --version                                                                                                                                                                                             (base) 
cargo-contract 0.16.0-unknown-x86_64-linux-gnu


```

Reply:
edited